src/activation.rs

```diff
@@ -211,15 +214,13 @@ pub async fn run_activation(
 }
 
 /// Get the environment variables that are statically generated from the project and the environment.
+/// This returns an IndexMap to make sure they get exported in the correct order.
```

Comment:
wdyt about opening the parenthesis and explaining what does it mean "in the correct order"?

Reply:
```suggestion
/// Returns IndexMap to stay sorted, as pixi should export the metadata before exporting variables that could depend on it. 
```

What about this?

Reply:
agree!